src/ui/components/SecondaryToolbox/NewConsole.tsx

```diff
@@ -1,8 +1,7 @@
 import { ExecutionPoint, PauseId } from "@replayio/protocol";
-import ConsoleRoot from "bvaughn-architecture-demo/components/console";
+import NewConsole from "bvaughn-architecture-demo/components/console";
```

Comment:
:question: Do we still need to call it `NewConsole`?

Reply:
Probably not :)